README.md

```diff
@@ -1,21 +1,22 @@
-# FAF
+# ABRT Analytics
 
 [![build status](https://copr.fedorainfracloud.org/coprs/g/abrt/faf-devel/package/faf/status_image/last_build.png)](https://copr.fedorainfracloud.org/coprs/g/abrt/faf-devel/package/faf/)
 
-**Originated as Fedora Analysis Framework.
-The ultimate tool to fix problems of application crashes.**
+**ABRT Analytics collects reports from collected by ABRT and aggregate them. Developer or DevOps admin can sort them using number of occurency and see all similar reports.**
```

Comment:
Fixed. Because this is our target group. We do not target normal or even a newbie user. Of course, anyone can send a report, but developers, power users, and admins are the majority - it is our target group.

Reply:
Right, the wording just seems strange to me, that only developers and such can do the sorting. I can make the assumption that it means those groups are our target audience, but that’s it.